doc/gui/examples/controls/column_name_styling.py

```diff
@@ -0,0 +1,21 @@
+# Example for column name styling
+
+from taipy.gui import Gui
+import pandas as pd
+from taipy.gui import Markdown
+
+# Sample data in DataFrame format
+df = pd.DataFrame({
+    "Name": ["Alice", "Bob", "Charlie"],
+    "Age": [25, 30, 35],
+    "Occupation": ["Engineer", "Doctor", "Artist"]
```

Comment:
@FredLL-Avaiga Please check now will the new changes would work ? or should i change it to job from occupation.

Reply:
You tell me.
the best thing is to try it :-D
move the example file at the root of your local repo and run it

Reply:
I changed the name of the column because I wanted to show how a column name with space translates to a class name.
I think this should be part of the example